torch/onnx/_internal/exporter/_core.py

```diff
@@ -434,6 +436,42 @@ def _handle_call_function_node_with_lowering(
             # use SequenceAt to get the value. This is handled by torchlib
             pass
 
+    if node.target.__class__.__name__ == "CondOp":
+        # TODO: The condition should be replaced by something more relevant.
+        # Controlflow is handled in the exporter and not in torchlib
+        # as subgraph are only available in this code.
+        cond, true_graph, false_graph, graph_args = node.args
+        assert (
+            true_graph.name in local_functions
+        ), f"Unable to find {true_graph.name!r} in local_functions={local_functions}"
+        assert (
+            false_graph.name in local_functions
+        ), f"Unable to find {false_graph.name!r} in local_functions={local_functions}"
+        # builds local functions
+        val_args = tuple(n.meta["val"] for n in graph_args)
+        # Expected `mod` to be an instance of `torch.nn.Module`, got <class 'torch.fx.graph.Graph'>.
+        # But this scenario should be supported.
+        true_program = torch.onnx.export(
+            local_functions[true_graph.name], val_args, dynamo=True, fallback=False
+        )
+        false_program = torch.onnx.export(
```

Comment:
I think we should use lower level (internal) api here? At least use _core.export. But maybe iterating graph function would be sufficient.

Reply:
It might be better to prevent any bug with initializers but I need first to see how to add a subgraphs in the IR. The exporter returns a model, not a subgraph, there is no subgraph except a list of onnx nodes.

Reply:
> I think we should use lower level (internal) api here? At least use _core.export. But maybe iterating graph function would be sufficient.

I don't well the internal API. Feel free to name one function.

Reply:
For new logic, I dont' know very well either. But I think it follows the main logic of the old one that we iterate the FX graph, so I guess there might be a big for loop function to generate the sub. Let me check.

Reply:
Right - we should not use the public api. Happy to discuss in chat.

Reply:
For creating functions, I would consider adapting `_add_nodes` to handle function creations. Either refactor it to handle functions, or take the for loop out and create a similar function to handle ONNX function creation.

Reply:
I used local functions.